build.gradle

```diff
@@ -151,66 +116,3 @@ gitProperties {
     customProperty 'buildDate', "${buildDate()}"
     customProperty 'develop', 'true'
 }
-
-launch4j {
-    mainClassName = "${railsMainClass}"
-    jar = "${buildDir}/libs/${project.name}-${project.version}-all.jar"
-    outfile = "${project.name}-${project.version}.exe"
-    icon = "../../${buildFilesDir}/${railsIcon}"
-    textVersion = {  project.version.replace('alpha','98.').replace('beta','99.').replace('RC','100.') }
-    copyright = "${railsCopyright}"
-    companyName = "${railsVendor}"
-    description = "${railsDescription}"
-    productName = "${project.name}"
-    internalName = "${project.name}"
-}
-
-macAppBundle {
-    mainClassName = "${railsMainClass}"
-    icon = "../../${buildFilesDir}/${railsIcon}"
-    javaProperties.put("apple.laf.useScreenMenuBar", "true")
-    //runtimeConfigurationName = 'shadow'
-    //jarTask = 'shadowJar'
-}
-
-ospackage {
-    release = '1'
-    os = 'LINUX'
-    packageGroup = 'Amusements/Games'
-    packageDescription = ''
-    requires('jdk', '1.8', GREATER | EQUAL)
-
-    from('src/scripts/rails-18xx') {
-        into '/usr/bin/'
-        fileMode 0555
-    }
-    from(jar.outputs.files) {
-        // TODO: needs to be the shadow jar
-        into '/usr/lib/rails-18xx/'
-    }
-    from('src/scripts/rails-18xx.desktop') {
-        into '/usr/share/applications/'
-    }
-    from('./src/main/resources/build/rails.png') {
-        into '/usr/share/pixmaps/'
-    }
-}
-
-//buildRpm.dependsOn shadowJar
-
-
-def isNonStable = { String version ->
-  def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
-  def regex = /^[0-9,.v-]+(-r)?$/
-  return !stableKeyword && !(version ==~ regex)
-//    def rejected = ['alpha', 'beta', 'rc', 'cr', 'm'].any { qualifier -> version ==~ /(?i).*[.-]${qualifier}[.\d-]*/ }
-//    return rejected
-}
-
-dependencyUpdates {
-  gradleReleaseChannel = 'current'
-  rejectVersionIf {
-    isNonStable(it.candidate.version)
-  }
-}
```

Comment:
This is part of the ben-manes plugin so think we should retain it.

Reply:
yes

Reply:
I have readded the `ben-manes` plugin